test/utils

```diff
@@ -121,6 +121,16 @@ assertFileExists() {
   assertTrue "File ${name} does not exist" "[ -f ${tgt} ]"
 }
 
+assertDirExists() {
+  local path=${1}
+  assertTrue "Dir ${path} does not exist" "[ -d ${path} ]"
```

Comment:
s/does not//
s/exist/exists/

Reply:
You found issue in the right direction but the one had issue is line 131 down there. I fixed it in https://github.com/heroku/heroku-buildpack-go/pull/245/commits/570e7964393f0464002ea59b059863c4c4b6d8e1